src/platform.ts

```diff
@@ -24,6 +24,9 @@ export enum Runtime {
     RHEL_7 = <any>'RHEL_7',
     Ubuntu_14 = <any>'Ubuntu_14',
     Ubuntu_16 = <any>'Ubuntu_16',
+    Ubuntu_18 = <any>'Ubuntu_18',

```

Comment:
I'd also suggest adding version-agnostic runtimes like RIDs : https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids

e.g. for Ubuntu you could have 

```
Ubuntu = <any>'Ubuntu',
Ubuntu_14 = <any>'Ubuntu_14',
Ubuntu_16 = <any>'Ubuntu_16',
Ubuntu_18 = <any>'Ubuntu_18',
Ubuntu_20 = <any>'Ubuntu_20',
Ubuntu_22 = <any>'Ubuntu_22',
```

That way if a consumer has a version-agnostic package they can just use that one and not have to arbitrarily choose a specific version.

Reply:
done.